SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -176,13 +174,12 @@ class USpatialReceiver : public UObject
 
 	AActor* FindSingletonActor(UClass* SingletonClass);
 
+	void OnHeartbeatComponentUpdate(Worker_ComponentUpdateOp& Op);
+
 public:
 	TMap<FUnrealObjectRef, TSet<FChannelObjectPair>> IncomingRefsMap;
 
 private:
-	template <typename T>
```

Comment:
Drive-by fix, this function has been removed a long time ago but declaration was still there.